lighthouse-core/config/no-throttling-config.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
is the intention for this to be for testing only? Consider moving to `lighthouse-cli/test/smokehouse/test-definitions/`?